cogs/modmail.py

```diff
@@ -1212,24 +1212,24 @@ async def logs_closed_by(self, ctx, *, user: User = None):
         session = EmbedPaginatorSession(ctx, *embeds)
         await session.run()
 
-    @logs.command(name="id")
+    @logs.command(name="key", aliases=["id"])
     @checks.has_permissions(PermissionLevel.SUPPORTER)
-    async def logs_id(self, ctx, key: str):
+    async def logs_key(self, ctx, key: str):
         """
-        Get the log link for the specified log ID
+        Get the log link for the specified log key
         """
         icon_url = ctx.author.avatar.url
 
         logs = await self.bot.api.find_log_entry(key)
 
-        if not any(not log["open"] for log in logs):
+        if not logs:
             embed = discord.Embed(
                 color=self.bot.error_color,
                 description=f"Log entry `{key}` not found.",
             )
             return await ctx.send(embed=embed)
 
-        logs = reversed(log for log in logs if not log["open"])
+        logs = list(reversed(logs))
```

Comment:
There should only be at most one log with a certain key, there's really no use for the extra `list` and `reversed`. I'd suggest removing this line.